acos_client/v30/vrrpa/vrid.py

```diff
@@ -38,25 +38,22 @@ def exists(self, vrid_val):
         except acos_errors.NotFound:
             return False
 
-    def _build_params(self, vrid_val, threshold=None, disable=None, floating_ip=None,
+    def _build_params(self, vrid_val, threshold=None, disable=None, floating_ips=[],
                       is_partition=False):
         vrid = {'vrid-val': vrid_val}
-
-        vrid_floating_ip = None
-        if floating_ip:
+        vrid_floating_ips = None
+        if floating_ips:
             if is_partition:
-                vrid_floating_ip = {
-                    'ip-address-part-cfg': [{
-                        'ip-address-partition': floating_ip
-                    }]
+                ip_partition_list = map(lambda ip: {'ip-address-partition': ip}, floating_ips)
```

Comment:
map function returns an iterator object for python 3, we will have to explicitly convert it to list. we can use list comprehension over here  like,
 [{'ip-address-partition': ip} for ip in floating_ips]

Reply:
That is very true. Fixed